### PR TITLE
fix(DB/Script): The Nexus updates

### PR DESCRIPTION
**Pull request by Kitzunu**
## CHANGES PROPOSED:
- Add money loot to all bosses that were missing. (These values do not have a reliable source. I used old videos and the already entered money values in DB to calculate something that looks okay)
- Berinand's Research respawn time changed to 2 sec
- Respawn Crazed Mana-Wyrm and fix weird hover state
- Respawn Azure Skyrazor and fix weird hover state
- Chaotic Rift rework SAI to be blizzlike.
- Respawn Crazed Mana-Wraith

## ISSUES ADDRESSED:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/2874

## TESTS PERFORMED:
- Ran SQL
- Tested in-game. Works!

## HOW TO TEST THE CHANGES:

- Check the linked issue

## Target branch(es):
- [x] Master

 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

**Comment by Maurowin** [comment]
All in all it's much better!
Humanoid dragons are supposed to kill spirits.
WoWScrnShot_041120_220908
WoWScrnShot_041120_220940

Dragons shall be already dead
WoWScrnShot_041120_221037

Missing mobs
WoWScrnShot_041120_221542

**Comment by Kitzunu** [comment]
@Maurowin All done!

**Comment by Maurowin** [comment]
Some bugs, i make a video with comment in :
https://youtu.be/8OoNPvIwjpk

**Comment by Kitzunu** [comment]
@Maurowin Okay all the issues should be fixed now.

However, the book I can't make visible all the time. It has a respawn of 2 seconds now at least, so it is as good as it gets

**Comment by Maurowin** [comment]
2sec is fine and all time visible too x) i will test soon

File: data/sql/updates/pending_db_world/rev_1586619822318777300.sql [hunk 1]
```diff
@@ -154,4 +167,52 @@ INSERT INTO `creature` (`guid`, `id`, `map`, `zoneId`, `areaId`, `spawnMask`, `p
...
+(126604, 26782, 576, 0, 0, 3, 1, 27469, 0, 453.272, -178.744, -14.0888, 5.79238, 3600, 1, 0, 37164, 0, 1, 0, 0, 0, '', 0),
+(126605, 26782, 576, 0, 0, 3, 1, 27469, 0, 247.919, -235.086, -8.48918, 2.34172, 3600, 1, 0, 37164, 0, 1, 0, 0, 0, '', 0),
+(126606, 26782, 576, 0, 0, 3, 1, 27469, 0, 251.404, -238.64, -8.25408, 2.34654, 3600, 1, 0, 37164, 0, 1, 0, 0, 0, '', 0),
+(126607, 26782, 576, 0, 0, 3, 1, 27469, 0, 403.166, -313.62, -14.0888, 0.612719, 3600, 1, 0, 37164, 0, 1, 0, 0, 0, '', 0),
+(126608, 26782, 576, 0, 0, 3, 1, 27469, 0, 397.014, -322.266, -14.0888, 0.174134, 3600, 1, 0, 37164, 0, 1, 0, 0, 0, '', 0),
+(126609, 26782, 576, 0, 0, 3, 1, 27469, 0, 431.531, -273.502, -14.0888, 2.62032, 3600, 1, 0, 37164, 0, 1, 0, 0, 0, '', 0),
+(1266090, 26782, 576, 0, 0, 3, 1, 27469, 0, 431.531, -273.502, -14.0888, 2.62032, 3600, 1, 0, 37164, 0, 1, 0, 0, 0, '', 0),
+(126440, 28231, 576, 0, 0, 3, 1, 27609, 1, 356.727, -174.203, -14.0888, 3.80708, 3600, 0, 0, 37164, 3231, 2, 0, 0, 0, '', 0),
+(126441, 28231, 576, 0, 0, 3, 1, 27609, 1, 383.407, -184.995, -14.0888, 2.74376, 3600, 0, 0, 37164, 3231, 2, 0, 0, 0, '', 0),
```
Comment (pklloveyou) [inline data/sql/updates/pending_db_world/rev_1586619822318777300.sql:202]: sql```
(126441, 28231, 576, 0, 0, 3, 1, 27609, 1, 383.407, -184.995, -14.0888, 2.74376, 3600, 5, 0, 37164, 3231, 1, 0, 0, 0, '', 0),
```

Comment (Kitzunu) [reply data/sql/updates/pending_db_world/rev_1586619822318777300.sql:202]: No, that is wrong. It should not have random movement

**Comment by Kitzunu** [comment]
@Maurowin Any updates? :)

**Comment by Maurowin** [comment]
this patch work in heroic mode? didn't test.

These mobs shall be like screenshot when he die and reset their aggro when he ressurect.
WoWScrnShot_042020_125140

**Comment by Kitzunu** [comment]
Okay, to make it easy on me. Since I managed to fuck up my fork.

Are all the changes done here good so far? Can we please open anything else in another ticket, since I cannot edit this PR?

**Comment by Maurowin** [comment]
I guess yes

**Comment by Kitzunu** [comment]
Okay, compile a list of other issues in Nexus in a new issue and I will take a look at it later.

Ready for Review! 🚀

**Comment by BarbzYHOOL** [comment]
check here guys https://github.com/azerothcore/azerothcore-wotlk/pull/2914

**Pull request closed without merging**